portal/src/it/scala/cromwell/pipeline/controller/ProjectControllerItTest.scala

```diff
@@ -0,0 +1,86 @@
+package cromwell.pipeline.controller
+
+import java.util.UUID
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import com.dimafeng.testcontainers.{ForAllTestContainer, PostgreSQLContainer}
+import com.typesafe.config.Config
+import cromwell.pipeline.ApplicationComponents
+import cromwell.pipeline.datastorage.dao.repository.utils.{TestProjectUtils, TestUserUtils}
+import cromwell.pipeline.datastorage.dto.{Project, ProjectDeleteRequest, ProjectId, ProjectUpdateRequest, UserId}
+import cromwell.pipeline.datastorage.utils.auth.AccessTokenContent
+import cromwell.pipeline.utils.TestContainersUtils
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport
+import org.scalatest.{AsyncWordSpec, Matchers}
+
+class ProjectControllerItTest
+  extends AsyncWordSpec
+    with Matchers
+    with ScalatestRouteTest
+    with PlayJsonSupport
+    with ForAllTestContainer {
+
+  override val container: PostgreSQLContainer = TestContainersUtils.getPostgreSQLContainer()
+  container.start()
+  implicit val config: Config = TestContainersUtils.getConfigForPgContainer(container)
+  private val components: ApplicationComponents = new ApplicationComponents()
+  override protected def beforeAll(): Unit = components.datastorageModule.pipelineDatabaseEngine.updateSchema()
+  private val ownerId = "1"
+  private val dummyUser = TestUserUtils.getDummyUser(ownerId)
+  private def projectId = ProjectId(UUID.randomUUID().toString)
+  import components.controllerModule.projectController
+  import components.datastorageModule.projectRepository
+  import components.datastorageModule.userRepository
```

Comment:
The same import used in UserControllerItTest. I was thinking it's OK.

Reply:
And because we using variable components, initialized on 26 line. I need this import, because I define a components variable into this class. I think my action is correct.